src/axom/quest/interface/inout.cpp

```diff
@@ -444,7 +575,30 @@ bool inout_evaluate(double x, double y, double z)
     return false;
   }
 
-  return s_inoutHelper.within(x, y, z);
+  const int dim = inout_get_dimension();
+  return (dim == 2) ? s_inoutHelper2D.within(x, y, z)
+                    : s_inoutHelper3D.within(x, y, z);
+}
+
+int inout_evaluate(const double* x, const double* y, int npoints, int* res)
+{
+  if(!inout_initialized())
+  {
+    SLIC_WARNING("quest inout query must be initialized "
```

Comment:
Error instead of warning?

Reply:
There are multiple more calls like this. If you change one, the rest should be made consistent.

Reply:
Same as above:
> For the inout "C" interface, we opted to have everything use return codes so the end user can decide how to do error handling.